<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iiClass</class>
 <widget class="QMainWindow" name="iiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ii</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_qlabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInteractive_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Image"/>
   <addaction name="actionSave_Image"/>
   <addaction name="action_forward"/>
   <addaction name="actionShow_camera"/>
   <addaction name="actionDesign"/>
   <addaction name="actionRotate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Image">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/folder.png</normaloff>:/image/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionInteractive_Image">
   <property name="text">
    <string>Interactive Image</string>
   </property>
  </action>
  <action name="action_forward">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/forward.png</normaloff>:/image/forward.png</iconset>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
   <property name="toolTip">
    <string>forward</string>
   </property>
  </action>
  <action name="actionShow_camera">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/camera.png</normaloff>:/image/camera.png</iconset>
   </property>
   <property name="text">
    <string>Show_camera</string>
   </property>
  </action>
  <action name="actionDesign">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/design.png</normaloff>:/image/design.png</iconset>
   </property>
   <property name="text">
    <string>design</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/rotate.png</normaloff>:/image/rotate.png</iconset>
   </property>
   <property name="text">
    <string>rotate</string>
   </property>
  </action>
  <action name="actionShadow">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/shadow.png</normaloff>:/image/shadow.png</iconset>
   </property>
   <property name="text">
    <string>shadow</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ii.qrc"/>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
